com.unity.robotics.urdf-importer/Tests/Runtime/UrdfComponents/UrdfJoints/UrdfJointContinuousTests.cs

```diff
@@ -0,0 +1,137 @@
+using NUnit.Framework;
+using UnityEngine;
+using RosSharp.Urdf;
+using Joint = RosSharp.Urdf.Joint;
+
+public class TestUrdfJointContinuous : UrdfJointContinuous
+{
+    public void TestImportJointData(Joint joint)
+    {
+        unityJoint = gameObject.GetComponent<ArticulationBody>();
+        ImportJointData(joint);
+    }
+
+    public Joint TestExportSpecificJointData(Joint joint)
+    {
+        unityJoint = gameObject.GetComponent<ArticulationBody>();
+        return ExportSpecificJointData(joint);
+    }
+
+    public void SetAxisOfMotion(Vector3 axisofMotion) => this.axisofMotion = axisofMotion;
+    public void Dynamics(Joint.Dynamics dynamics) => SetDynamics(dynamics);
+}
+
+public class UrdfJointContinuousTests
+{
+    [Test]
+    public void Create_Succeeds()
+    {
+        GameObject linkObject = new GameObject("link");
+        UrdfJoint joint = UrdfJointContinuous.Create(linkObject);
+        UrdfJoint urdfJoint = linkObject.GetComponent<UrdfJoint>();
+        ArticulationBody articulationBody = linkObject.GetComponent<ArticulationBody>();
+
+        Assert.IsTrue(joint is UrdfJointContinuous);
+        Assert.AreEqual(joint, urdfJoint);
+        Assert.AreEqual(ArticulationJointType.RevoluteJoint, articulationBody.jointType);
+
+        Object.DestroyImmediate(linkObject);
+    }
+
+    [Test]
+    public void GetPositionVelocityEffort_Succeeds()
+    {
+        GameObject baseObject = new GameObject("base");
+        GameObject linkObject = new GameObject("link");
+        linkObject.transform.parent = baseObject.transform;
+
+        UrdfJoint.Create(baseObject, UrdfJoint.JointTypes.Fixed);
+        UrdfJoint joint = UrdfJointContinuous.Create(linkObject);
+        ArticulationBody articulationBody = linkObject.GetComponent<ArticulationBody>();
+        articulationBody.jointPosition = new ArticulationReducedSpace(1, 2, 3);
+        articulationBody.jointVelocity = new ArticulationReducedSpace(4, 5, 6);
+        articulationBody.jointForce = new ArticulationReducedSpace(7, 8, 9);
+
+        Assert.AreEqual(1, joint.GetPosition());
+        Assert.AreEqual(4, joint.GetVelocity());
+        Assert.AreEqual(7, joint.GetEffort());
+
+        Object.DestroyImmediate(baseObject);
+        Object.DestroyImmediate(linkObject);
```

Comment:
question: I always thought destroying a parent would automatically destroy the children--would this cause any issues? and/or am I mistaken and should update my unit tests to also destroy children individually?

Reply:
no, you are right that the destroy method will remove the children. I didn't realize it. Will update. Thanks!